frontend/src/scenes/insights/InsightTabs/RetentionTab.tsx

```diff
@@ -118,7 +125,7 @@ export function RetentionTab(): JSX.Element {
             </Button>
             <ActionFilterDropdown
                 open={returningOpen}
-                logic={entityLogicReturning}
+                logic={entityLogicReturning as any}
```

Comment:
Not sure if type here should be kea logic

Reply:
I tried this but doesn't work because ActionFilterDropdown expects something else. See this thread on slack: https://posthog.slack.com/archives/C0113360FFV/p1614605023002800